Gems/PhysX/Code/Include/PhysX/Joint/PhysXJointBus.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+
+#pragma once
+
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/std/utils.h>
+#include <AzCore/Math/Transform.h>
+
+namespace PhysX
+{
+    //! Interface to communicate with PhysX joint's motor.
+    class JointInterface : public AZ::EBusTraits
```

Comment:
update: Please name this `JointInterfaceRequests` (or maybe just `JointRequests`) to be more consistent with other EBus naming across the engine

Reply:
I vote for `JointRequests`.

Reply:
Yeah, I would avoid having Interface in the name because it might get confusing with AZ::Interface. If it's specifically for joint drives, it might be good to go with something like JointDriveRequests.

Reply:
Set to "JointRequests "

Reply:
JointRequests will be.